public/css/list/item-list.less

```diff
@@ -58,12 +58,8 @@
 
     .caption {
       height: 3em;
-      text-overflow: ellipsis;
-      overflow: hidden;
 
-      .plugin-output {
-        .line-clamp();
-      }
```

Comment:
If the line clamp is now not (only) applied to the plugin output anymore, please also adjust the reset on line 90 so that it has an effect again.

Reply:
I think, we can remove the `line-clamp("reset")` property from line 88, since we always have a max no of lines to display, don't we?

It's just 4 in .detailed instead of 2.